index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="logo">
+      <p class="logo-text">My Logo</p>
+    </div>
+    <div class="menu-container">
+      <div class="menu">
+        <img src="./images/Union.png" alt="menu" class="ham-menu">
+      </div>
+    </div>
+  </nav>
+  <section class="main-container">
+    <div class="about-container">
+      <h1 class="about">
+        I'm Isai. <br>
+        Glad to see you!
+      </h1>
+    </div>
+    <div class="description-container">
+      <p class="description">
+        I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+    </div>
+    <div class="supporting-text-container">
+      <p class="suporting-text">
+        LET'S CONNECT
+      </p>
+    </div>
+    <div class="social-media-container">
+      <a class="twitter" href="#"><img src="./images/twitter.svg" alt="" class="social"></a>
+      <a class="github" href="#"><img src="./images/github.svg" alt="" class="social"></a>
+      <a class="mail" href="#"><img src="./images/mail.svg" alt="" class="social"></a>
+      <a href="#"><img src="./images/other.svg" alt="" class="social"></a>
+      <a class="Linkedin" href="#"><img src="./images/linkedin.svg" alt="" class="social"></a>
```

Comment:
The social media icons should have more space between them in look-alike as the template